.cloudbuild/run_deploy.sh

```diff
@@ -0,0 +1,18 @@
+# /bin/bash
+
+gcloud run deploy ${_SERVICE} \
+    --platform managed --region ${_REGION} \
+    --image gcr.io/$PROJECT_ID/${_SERVICE}  \
+    --allow-unauthenticated  \
+    --add-cloudsql-instances ${_DATABASE_INSTANCE} \
+    --service-account cloudrun-berglas-python@${PROJECT_ID}.iam.gserviceaccount.com \
+    --update-env-vars DATABASE_URL=berglas://${_BERGLAS_BUCKET}/database_url,SECRET_KEY=berglas://${_BERGLAS_BUCKET}/secret_key,GS_BUCKET_NAME=berglas://${_BERGLAS_BUCKET}/media_bucket
```

Comment:
Note for readability you can do this and `gcloud` will concatenate them:

```
--update-env-vars DATABASE_URL=berglas://${_BERGLAS_BUCKET}/database_url
--update-env-vars SECRET_KEY=berglas://${_BERGLAS_BUCKET}/secret_key
--update-env-vars GS_BUCKET_NAME=berglas://${_BERGLAS_BUCKET}/media_bucket
```

Reply:
For some reason I didn't realise this was possible! 🤯 Such readability!

Reply:
Oh TIL as well